share/completions/ln.fish

```diff
@@ -1,16 +1,16 @@
+complete -c ln -s s -l symbolic            -d 'Make symbolic links instead of hard links'
+complete -c ln      -l backup              -d 'Make a backup of each existing destination file' -a "none off numbered t existing nil simple never"
+complete -c ln -s b                        -d 'Make a backup of each existing destination file'
+complete -c ln -s dF -l directory          -d 'Allow superuser to attempt to hard link directories'
```

Comment:
Ahh! I think it's less hassle to just remove it. I doubt it's useful to anyone, just noticed it was missing and thought "why not" add it.

Commit updated with `-F` omitted.